.eslintrc

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
where and why is this file used?

Reply:
Builder has this file in the repo, so I thought it is kind of good practice.

Reply:
Builder uses eslint for parsing expressions, that's why it's config is there. Initially I was surprised that you used it but then I realized that it's some legacy left-over to be cleaned up.

Reply:
Actually I use, it has some differences from builder config